cookbooks/blogs/recipes/default.rb

```diff
@@ -43,14 +43,15 @@
 git "/srv/blogs.openstreetmap.org" do
   action :sync
   repository "git://github.com/gravitystorm/blogs.osm.org.git"
+  depth 5
   user "blogs"
   group "blogs"
   notifies :run, "execute[/srv/blogs.openstreetmap.org/Gemfile]", :immediately
 end
 
 execute "/srv/blogs.openstreetmap.org/Gemfile" do
   action :nothing
-  command "bundle install"
+  command "bundle install --deployment"
```

Comment:
If you're doing this then it can run as user `blobs` instead of root.

Reply:
Done.